backend/pyproject.toml

```
@@ -52,7 +52,7 @@ cryptography = "<2.6"
 flask-kvsession = {git = "https://github.com/inspirehep/flask-kvsession.git"}
 python-redis-lock = "^3.3"
 # FIXME: orcid tests are failing with `4.4` because of OrderedDict changes
-lxml = "~=4.3,<4.4"
+lxml = "4.4"
```

Is 4.5 incompatible?
```suggestion
lxml ~= "4.4"
```
So we could at least update to latest fix on 4.4. branch.

---

Ok, I think that it should (there are no significant changes in 4.5 comparing to 4.4, just some bug fixes and one new feature)